bool     advanced_mode
float64  max_pressure
float64  min_pressure
duration timeout
